scripts/output/projects/FABLE_India_subnational_validation.R

```diff
@@ -0,0 +1,105 @@
+# |  (C) 2008-2021 Potsdam Institute for Climate Impact Research (PIK)
+# |  authors, and contributors see CITATION.cff file. This file is part
+# |  of MAgPIE and licensed under AGPL-3.0-or-later. Under Section 7 of
+# |  AGPL-3.0, you are granted additional permissions described in the
+# |  MAgPIE License Exception, version 1.0 (see LICENSE file).
+# |  Contact: magpie@pik-potsdam.de
+
+# --------------------------------------------------------------
+# description: India subnational validation 
+# comparison script: FALSE
+# ---------------------------------------------------------------
+
+#Version 1.00 - Miodrag Stevanovic, Prantika Das
+# 1.00: first working version
+
+library(lucode2)
+library(magpie4)
+library(madrat)
+library(mrfable)
+library(mip)
+library(mrvalidation)
+library(magpiesets)
+library(lusweave)
+
+print("Start India state level validation output runscript")
+
+############################# BASIC CONFIGURATION #######################################
+
+if(!exists("source_include")) {
+  
+  outputdir <- "output//v4p77/h12_2022-12-15_22.09.46"
+  
+  ###Define arguments that can be read from command line
+  readArgs("outputdir")
+}
+# hist     <- "runs/validation.mif"
+file    <- file.path(outputdir,paste0("India_subnational_validation_",format(Sys.time(), "%Y%H%M%S"),".pdf"))
+#########################################################################################
+
+print(paste0("Script started for output directory: ",outputdir))
+
+# Determine cells for India
+mapping <- toolGetMapping("scripts/npi_ndc/policies/country2cell.rds")
+mapping$iso_mag <- paste(mapping$iso,mapping$cell,sep=".")
+# load Indian states
+ind_states <- read.csv(system.file("extdata", "regional/india_state_code.csv", package = "mrfable"))
```

Comment:
I'd suggest that we stick to camelCase in magpiemodel as well, so it is consistent with our variable naming style in R packages.
```suggestion
indStates <- read.csv(system.file("extdata", "regional/india_state_code.csv", package = "mrfable"))
```

Reply:
No, both functions and variables should follow camelCase